format/FlightSql.proto

```diff
@@ -908,30 +908,30 @@ enum Nullable {
   /**
    * Indicates that the fields does not allow the use of null values.
    */
-  TYPE_NO_NULLS = 0;
+  NULLABILITY_NO_NULLS = 0;
 
   /**
    * Indicates that the fields allow the use of null values.
    */
-  TYPE_NULLABLE = 1;
+  NULLABILITY_NULLABLE = 1;
 
   /**
    * Indicates that nullability of the fields can not be determined.
    */
-  TYPE_NULLABLE_UNKNOWN = 2;
+  NULLABILITY_UNKNOWN = 2;
```

Comment:
Do ODBC/JDBC equate UNKNOWN to the value 2 specifically? If not, we should try to follow Protobuf convention and put UNKNOWN as 0.

Reply:
Yes, both UNKNOW are related to the value 2 in JDBC and ODBC